src/articles/components/ArticleView/index.tsx

```diff
@@ -37,6 +37,8 @@ export const ArticleView = ({ articleId }: IProps) => {
 
   return (
     <div className={style.container}>
+      {/*
+      // @ts-ignore-next-line TS2604 */}
```

Comment:
I can confirm this still works, even in SSR, but Typescript is not happy for some reason, so this was added, it is likely we can remove it at some point in the near future, but unsure about what caused it in the first place.